exercise10.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+def get_price_quantity():
+    """Get price and quantity from user"""
+
+    price = float(input("Enter the price:$"))
+    quantity = int(input("Enter the quantity: "))
+
+    return price, quantity
+
+
+def get_subtotal(price, quantity):
+    """Calculate subtotal: price * quantity"""
+
+    return price * quantity
+
+
+def calculate_sum_of_subtotals(subtotal_item1, subtotal_item2, subtotal_item3):
+    """Add all subtotals together to get sum of subtotals"""
+
+    return subtotal_item1 + subtotal_item2 + subtotal_item3
+
+
+def convert_tax_to_decimal(TAX_RATE):
```

Comment:
@paulghaddad   I could call convert_tax_to_decimal and add it to my parameter, instead of re-calculating it below, right?

Reply:
Not sure what you mean. However, remember `TAX_RATE` on line 30 is the _parameter_, not the specific argument you pass in. It is the general name of the input for the function. Thus, it should be lowercase here. But when you _call_ the function, you can pass in any named variable as the `argument`. Which is what you do in the `main` file:

https://github.com/amymhaddad/version_2_exercises_for_programmers/pull/35/files#diff-98e5bbb0f3420abac5ca67b81848262aR17

or in the test:

https://github.com/amymhaddad/version_2_exercises_for_programmers/pull/35/files#diff-322cd8d0a431861e5483d2fd42dd5905R39